client/src/plugins/version-info/ReleaseInfo.js

```diff
@@ -44,28 +44,35 @@ export function ReleaseInfo(props) {
     <div className={ css.ReleaseInfo }>
       <ul className="dashed">
         <li>
-          <h4>Support for Camunda Cloud 1.4</h4>
-          With this release, we added support for Camunda Cloud 1.4.<br />
-          You can now evaluate a DMN decision in a Business Rules Task.
-          Specify the called decision (DMN) by its ID and assign the result to a variable via the properties panel.
-          Learn more about this feature in the <a href="https://docs.camunda.io/docs/components/modeler/bpmn/business-rule-tasks/business-rule-tasks/">Camunda Cloud docs on Business Rule Tasks.</a>
+          <h4>Support for Camunda 8</h4>
+          With this release, we added support for Camunda 8.<br />
+          You can now model BPMN, and DMN diagrams as well as Forms for Camunda 8.
+          Using plugins, you can now also customize the Modeler specifically for Camunda 8.
+          We will now also use a more concise naming: <i>Camunda Platform</i> becomes Camunda 7
+          and <i>Camunda Cloud</i> becomes Camunda 8.<br />
+          Learn more about Camunda 8 in general <a href="toBeAdded">in the Camunda 8 blogpost.</a>
         </li>
         <li>
           <h4>Improved overall UI to become a "Visual IDE"</h4>
-          New welcome screen allows to easily create Platform and Cloud diagrams and forms.
-          The tab bar displays icons according to the file type and provides a new file menu via the plus button.
-          The properties panel has been completely redesigned. Tabs have been replaced with sections which can be opened and closed individually.
-          A granular breakdown with clear titles allows to identify relevant topics at a glance.
+          We re-worked the entire Modeler, including welcome screen, properties panel, status bar, tab bar, and bottom panel
+          following our visions of a "Visual IDE".<br />
+          Learn more about the changes <a href="https://camunda.com/blog/2022/01/camunda-modeler-5-0-0-alpha-0-released/">in this blog post</a>.
+        </li>
+        <li>
+          <h4>Process Debugging for Camunda 8 BPMN diagrams</h4>
+          The Modeler will now support you when automating BPMN processes with Camunda 8 by showing implementation errors,
+          and giving hints how to fix them.
         </li>
         <li>
           <h4>Support drilldown into collapsed subprocesses</h4>
```

Comment:
Any further read on this here that we could link?

Reply:
Just the blog post... (Which is alraedy linked per default from the whatsNewComponent)